public/app/features/dashboard-scene/scene/DashboardScene.test.tsx

```diff
@@ -315,6 +328,20 @@ function buildTestScene(overrides?: Partial<DashboardSceneState>) {
             $data: new SceneQueryRunner({ key: 'data-query-runner2', queries: [{ refId: 'A' }] }),
           }),
         }),
+        new SceneGridItem({
+          body: new VizPanel({
+            title: 'Panel with caching options',
+            key: 'panel-3',
+            pluginId: 'table',
+            $behaviors: [
+              new PanelQueryCachingOptionsBehavior({
+                cacheTimeout: '60',
+                queryCachingTTL: 200000,
```

Comment:
@dprokop Is there a good reason why these options are not part of SceneQueryRunner?

Reply:
@torkelo  not really. But I'm sceptical on putting non-oss (`queryCachingTTL`) options in the library... Same with `cacheTimeout` which is used by Graphite data source only.

Reply:
@dprokop but they are part of DataQueryRequest so I think it's fine, and I think scene apps would like to be able to use query caching as well

Reply:
Agree with `queryCachingTTL` which is a cross-data source option and makes sense in the apps. But I think I would stick with the core-only behavior for the Graphite's `cacheTimeout`. Feels non-desirable to me to put data source-specific API into the library.

Reply:
Or maybe I'm wrong and there are more data sources that use `cacheTimeout`...? I guess some OSS ones may use it.

Reply:
@dprokop it's part of plugin.json data source feature definition, I know zabbix uses it. A bit sad that the enterprise cachingTTL did not use that feature and built a separate option

Reply:
kk, then let's move it to SQR :)